docs/wiki/99_Developers/Developers_Guide.md

```diff
@@ -0,0 +1,215 @@
+
+Here is a basic workflow if you are trying to make a PR off of the _main_ branch:
+
+    git fetch aces
+    git checkout aces/main 
+    git checkout -b "branch_name"
+    vi filename
+    git add filename 
+    git status 
+    git commit -m "Initial commit"
+    git push my_fork branch_name
+
+There is usually no need to fully update your fork to the newest aces/LORIS version, since you can work off whichever branch you just fetched and checked-out.
+
+ **Never** push to aces/Loris. Your branch will be deleted. Always push to your fork, and then create the PR from GitHub.
+
+#### Branch Naming
+Follow the convention most LORIS developers use: “date-title...”.   
+Starting with the date allows branches to sort and display in order, and also helps sort out any conflicts faster in the main Loris repository.
+
+### Helpful commands to run before pushing new code:
+Travis is our continuous integration which runs on every PR in GitHub. These commands can catch small formatting errors that will cause the Travis build to fail.
+
+    make checkstatic  # includes these 3 commands: 
+                      # npm run lint:php, npm run lint:javascript, vendor/bin/phan
+
+    phpcs  modules/candidate_list/php/candidate_list.class.inc
+                      # phpcs allows you to specifically check the formatting of one file 
+
+    vendor/bin/phpcbf --standard=test/LorisCS.xml modules/candidate_list/php/candidate_list.class.inc
+                      # phpcbf allows you to automatically fix the errors 
+                      # found by phpcs marked with a check.
+
+Running unit or integration tests (useful if making significant changes to any module). More details can be found [here](../../../test/README.md):
+         
+    npm run tests:unit
+    # The commands below allow you to run unit tests for a specific 
+    # module or a specific test in a module
+    npm run tests:unit -- --filter CandidateTest
+    npm run tests:unit -- --filter CandidateTest::someTest
+    npm run tests:integration
+
+### Good practice for Travis CI
+As you get more comfortable with commits and PRs, to save time and Travis errors you can configure your setup so that `make checkstatic` must complete successfully 
+before any git push can be done. To do this, run 
+
+    git config core.hooksPath .githooks (git v2.9 or up)
+    Or 
+    cp .githooks/pre-push .git/hooks/pre-push 
+
+in the LORIS directory to configure the project. Make sure `.git/hooks/pre-push` is executable.
+
+**Notes:** Hooks can be skipped with the flag `--no-verify: git push fork --no-verify`.  
+Setting up an ssh key can also be useful to avoid the headache of entering your username and password on each push. 
+
+## 2. Issues
+Issues are used to bring the team's attention to any bugs or feature requests that you come across. When you make an issue, there are 4 options: bug fix, feature request, security 
```

Comment:
Should we remove this line break?

Reply:
@laemtl Which line break exactly? The one before the "Issues" header?

Reply:
Between security and vulnerability. 
![Screenshot from 2020-08-25 15-26-38](https://user-images.githubusercontent.com/32041423/91218696-73089e80-e6e7-11ea-972f-c0a9d0c5b799.png)